### ISSUE TITLE
Players qty (min/max) in room

### ISSUE THREAD

**Issue by fatal10error** [StepanoidTeam/PowerGridApi#2]
Min users in room 2 and max (for default map) is 6 (actually according to map)

**Comment by bobanko** [comment]
You mean The ruke that we check before start, right?

**Comment by bobanko** [comment]
You mean The rule that we check before start, right?

**Comment by fatal10error** [comment]
Yes (minimum), and rule we check when trying to join into room (maximum)
